bravado/fido_client.py

```diff
@@ -1,19 +1,15 @@
 # -*- coding: utf-8 -*-
 import logging
-
+import requests
 import six
 
 if six.PY3:
     raise ImportError("The fido client is not yet supported in py3")
 
 import fido
-from bravado_core.param import stringify_body as param_stringify_body
```

Comment:
Can you verify nothing else in bravado or bravado-core uses `bravado_core.param.stringify_body`? If not, we can ticket for removal.